README.md

```diff
@@ -1,65 +1,62 @@
-# plastic-scm README
+# Plastic SCM integration with VS Code's SCM features
 
-This is the README for your extension "plastic-scm". After writing up a brief description, we recommend including the following sections.
+## Intro
 
-## Features
-
-Describe specific features of your extension including screenshots of your extension in action. Image paths are relative to this README file.
-
-For example if there is an image subfolder under your extension project workspace:
-
-\!\[feature X\]\(images/feature-x.png\)
-
-> Tip: Many popular extensions utilize animations. This is an excellent way to show off your extension! We recommend short, focused animations that are easy to follow.
+[plasticscm](https://marketplace.visualstudio.com/items?itemName=Plastic.vscode-plasticscm) is a Visual Studio Code extension that integrates [Plastic SCM](https://www.plasticscm.com/). With this plugin, you can use Plastic SCM as your SCM tool. It is powered by [Codice Software](https://www.plasticscm.com/).
```

Comment:
These lines are too long! Set your markdownlint configuration to warn you about lines longer than 80 characters.

Reply:
Done! You can see my last commit.